npm/util/sysinfo_linux.go

```diff
@@ -0,0 +1,22 @@
+package util
+
+// this file has bits copied from github.com/zcalusic/sysinfo (v1.1.2)
+
+import (
+	"os"
+	"strings"
+)
+
+func KernelRelease() string {
+	return slurpFile("/proc/sys/kernel/osrelease")
+}
+
+// Read one-liner text files, strip newline.
+func slurpFile(path string) string {
+	data, err := os.ReadFile(path)
+	if err != nil {
+		return ""
+	}
+
+	return strings.TrimSpace(string(data))
+}
```

Comment:
I feel like we should inline this and handle the error. Weird things can happen (/proc gets unmounted or something) and it's nice to have the error.